routers/comments.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express')
+let db = require('../data/db')
+
+const router = express.Router({
+    mergeParams: true
+})
+
+// 	_______CRUD_________
+
+
+//______________________________________________________________
+// get - (http://localhost:4000/api/posts/:id/comments)
+router.get('/', (req, res) => {
+    const id = req.params.id;
+
+    db.findPostComments(id)
+        .then(data => {
+            res.json(data)
+        })
+        .catch(err => {
+            res.status(500).json({
+                error: 'The post with the specified ID does not exist.'
+            })
+        })
+})
+
+
+//______________________________________________________________
+// get (http://localhost:4000/api/posts/:id/comments/:commentId)
+router.get('/:commentId', (req, res) => {
+    db.findCommentById(req.params.id, req.params.commentId)
```

Comment:
`findCommentById` only takes in 1 parameter and that is the commentId.  If you pass in 2, it is only reading the first one which is the post id